helm/prometheus-rules/values.yaml

```diff
@@ -2,8 +2,8 @@ name: prometheus-rules
 namespace: monitoring
 serviceType: managed
 project:
-  branch: '[[ .Branch ]]'
-  commit: '[[ .SHA ]]'
+  branch: 'labels-matching'
+  commit: 'b8922e8fb6802e25942b60693f0e224fad869e12'
```

Comment:
Did you commit those on purpose?

Reply:
Absolutely not ! I don't even know when those changes appeared

Reply:
Edit: I forgot to `make clean` after running local tests